test/assertions.js

```diff
@@ -7,6 +10,17 @@ let SHIFT_SPEC = require('shift-spec').default;
 let Parser = require('../dist/parser').GenericParser;
 let EarlyErrorChecker = require('../dist/early-errors').EarlyErrorChecker;
 
+let expectationsDir = 'node_modules/shift-parser-expectations/expectations';
+
+
+function checkPassNotExists(program, isModule) {
+  let digest = crypto.createHash('sha256').update(normalize(program, isModule)).digest('hex');
```

Comment:
Extract this and the substring to its own function.

Reply:
Done, though it seems slightly silly.